React/Views/RCTViewManager.m

```diff
@@ -452,6 +454,8 @@ - (RCTShadowView *)shadowView
 RCT_EXPORT_VIEW_PROPERTY(onDragLeave, RCTDirectEventBlock)
 RCT_EXPORT_VIEW_PROPERTY(onDrop, RCTDirectEventBlock)
 #endif // ]TODO(macOS ISS#2323203)
+RCT_EXPORT_VIEW_PROPERTY(onKeyDown, RCTBubblingEventBlock) // macOS keyboard events
+RCT_EXPORT_VIEW_PROPERTY(onKeyUp, RCTBubblingEventBlock) // macOS keyboard events
```

Comment:
Shouldn’t these be up inside the `TARGET_OS_OSX` conditional?

Reply:
I was sort of on the fence on exposing them as we'll at some point need iOS keyboard support, but it's probably easier to just keep it consistent for now and move them inside